### DISCUSSION TITLE
СНЕЖКИ!

### DISCUSSION THREAD

**Discussion by delingar** [octothorp-team/public-dobrograd#753] [Предложения]
На горизонте уже видна зима и зимние карты серверов с новогодними ивентиками. Но есть игроки, которым не хватает этого, чтобы проникнутся зимней атмосферой. 
Предлагаю перенести с Платтсбурга механику игры в снежки и расставить по городу точки для их сбора (кто понимает, тот понимает). Это даст игрокам новое развлечение, способ отдохнуть от обыденности реального мира, а также улучшит зимнюю атмосферу на сервере

**Comment by VaR1aN** [comment]
Полностью согласен! Идея крутая, это добавит разнообразия в развлечениях, ну и еще будет круто провоцировать нигеров снежками :)
